po/common/common.pot

```diff
@@ -2234,8 +2234,7 @@ msgstr ""
 
 msgctxt "search_graph_blog"
 msgid ""
-"<p>→ learn more about <<site_name>> graphs: <a href=\"https://fr.blog."
-"openfoodfacts.org/news/des-graphiques-en-3-clics\">Graphs in 3 clicks</a> "
+"<p>→ learn more about <<site_name>> graphs: <a href=\"https://world.openfoodfacts.org/graphs-in-3-clicks\">Graphs in 3 clicks</a> "
```

Comment:
```suggestion
"<p>→ learn more about <<site_name>> graphs: <a href=\"/graphs-in-3-clicks\">Graphs in 3 clicks</a> "
```

Reply:
@hangy @stephanegigandet it's invoked from : https://world.openfoodfacts.org/cgi/product.pl?type=edit&code=4018077604218